streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -527,8 +528,11 @@ public void run() {
      * {@link TopologyBuilder} and {@link KStreamBuilder}).
      *
      * @return A string representation of the Kafka Streams instance.
+     *
+     * @deprecated Use the new method {@link #localThreadsMetadata()} to retrieve information.
```

Comment:
Nit: Maybe `Use {@link #localThreadsMetadata()} to retrieve runtime information.`